lib/Auth/Process/Statistics.php

```diff
@@ -19,6 +24,12 @@ public function process(&$request)
     {
         $dateTime = new DateTime();
         $dbCmd = new DatabaseCommand();
-        $dbCmd->insertLogin($request, $dateTime);
+        try {
+            $dbCmd->insertLoginFromFilter($request, $dateTime);
+        } catch (Exception $ex) {
```

Comment:
Which exception are you trying to catch, this one? https://github.com/dBucik/proxystatistics-simplesamlphp-module/blob/9c21fbb9ab6030af182bf02dc43b41809c0737c6/lib/DatabaseCommand.php#L396